### ISSUE TITLE
Internal Server Error when creating a fetched account

### ISSUE THREAD

**Issue by fxnn** [Mailu/Mailu#910]
When clicking the button to add an account for fetchmail, I get a `500 Internal Server Error`.

Below is the relevant log excerpt. All mailu docker images have the `1.6` label.

Thanks in advance for your support and for developing this great mail app!

```
admin_1      | [2019-02-04 22:42:02,717] ERROR in app: Exception on /ui/fetch/create/user@example.com [GET]
admin_1      | Traceback (most recent call last):
admin_1      |   File "/usr/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
admin_1      |     response = self.full_dispatch_request()
admin_1      |   File "/usr/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request
admin_1      |     rv = self.handle_user_exception(e)
admin_1      |   File "/usr/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception
admin_1      |     reraise(exc_type, exc_value, tb)
admin_1      |   File "/usr/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
admin_1      |     raise value
admin_1      |   File "/usr/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
admin_1      |     rv = self.dispatch_request()
admin_1      |   File "/usr/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
admin_1      |     return self.view_functions[rule.endpoint](**req.view_args)
admin_1      |   File "/usr/lib/python3.6/site-packages/flask_login/utils.py", line 261, in decorated_view
admin_1      |     return func(*args, **kwargs)
admin_1      |   File "/app/mailu/ui/access.py", line 28, in wrapper
admin_1      |     return callback(function, args, kwargs, dargs, dkwargs)
admin_1      |   File "/app/mailu/ui/access.py", line 19, in callback
admin_1      |     return function(*args, **kwargs)
admin_1      |   File "/app/mailu/ui/views/fetches.py", line 25, in fetch_create
admin_1      |     form.pw.validators = [wtforms.validators.DataRequired()]
admin_1      | AttributeError: 'FetchForm' object has no attribute 'pw'
admin_1      | 10.117.0.2 - - [04/Feb/2019:22:42:02 +0000] "GET /ui/fetch/create/user@example.com HTTP/1.0" 500 291 "http://mailu_front_1.docker.example.com/admin/ui/fetch/list" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.98 Safari/537.36"
```

**Comment by fxnn** [comment]
Looks like this is fixed in commit  34b3172 and will therefore be contained in 1.6.1. Looking forward to that release!

**Comment by muhlemmer** [comment]
Backport will be merged soon: #896
